Skeleton for backend and frontend communication

VincentPheng opened the pull request:
Fixed CSS.

VincentPheng committed:
Added skeleton to incorporate backend and database

VincentPheng committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into master
Skeleton for adding backend

VincentPheng committed:
Changed 1 line of css

VincentPheng committed:
Added skeleton for frontend-backend communication.
Fixed CSS.

HariCodesThings commented on frontend/src/components/Signup.js:
```diff
@@ -5,50 +5,85 @@ class Signup extends Component {
+                                    <input type="radio" id="driver" name="volunteer" value="driver" onChange={this.changeVolunteerType}/>
+                                    <label for="driver">Driver</label>
+                                    <input type="radio" id="other" name="volunteer" value="other" onChange={this.changeVolunteerType}/>
+                                    <label for="other">Other/Cook</label>
```
Change to "Kitchen Volunteer" instead of "Other/Cook"

EmilyGavrilenko commented on frontend/src/components/Signup.js:
```diff
@@ -5,50 +5,155 @@ class Signup extends Component {
+
+    changeVolunteerType = (event) => {
+        let volunteerData = this.state.volunteerData;
+        if (event.target.value === "driver") {
```
A user should have the option to be both a driver and a kitchen staff. It's not exclusive

EmilyGavrilenko commented on frontend/.eslintcache:
```diff
@@ -1 +1 @@
-[{"C:\\Users\\Vincent\\Documents\\H4I Bootcamp\\meals-that-connect-master\\frontend\\src\\index.js":"1","C:\\Users\\Vincent\\Documents\\H4I Bootcamp\\meals-that-connect-master\\frontend\\src\\App.js":"2","C:\\Users\\Vincent\\Documents\\H4I Bootcamp\\meals-that-connect-master\\frontend\\src\\components\\Login.js":"3","C:\\Users\\Vincent\\Documents\\H4I Bootcamp\\meals-that-connect-master\\frontend ...
+[{"C:\\Users\\Vincent\\Documents\\H4I Bootcamp\\meals-that-connect-master\\frontend\\src\\index.js":"1","C:\\Users\\Vincent\\Documents\\H4I Bootcamp\\meals-that-connect-master\\frontend\\src\\App.js":"2","C:\\Users\\Vincent\\Documents\\H4I Bootcamp\\meals-that-connect-master\\frontend\\src\\components\\Login.js":"3","C:\\Users\\Vincent\\Documents\\H4I Bootcamp\\meals-that-connect-master\\frontend ...
```
.eslintcache should be in your gitignore

EmilyGavrilenko committed:
Delete .eslintcache

I added this file to the .gitignore, once you pull the latest changes it shouldn't appear in your future commits

EmilyGavrilenko reviewed (approved):
Comment: a volunteer should have the option to be both a driver and a kitchen staff.
